Assets/Scripts/PlayerController.cs

```diff
@@ -55,68 +65,83 @@ public enum PlayerStateEnum
         RightWallSlide,
         Dead,
         Swing,
+        Dash
     }
-    
-    public PlayerStateEnum PlayerState { get; private set; }
-    
+
+    // TODO: PlayerState set should be a function that fires an action
+    public PlayerStateEnum PlayerState
+    {
+        get => _playerState;
+        private set
+        {
+            Debug.Log($"PlayerState: {value}");
```

Comment:
Can we add a checkbox so we can turn off these logs in the Unity UI?

Reply:
added